app/views/users/new.html.erb

```diff
@@ -0,0 +1,53 @@
+<h1>Create a new user</h1>
+
+<% if @user.errors.any? %>
+    <ul>
+        <% @user.errors.full_messages.each do |msg| %>
+            <li><%= msg %></li>
+        <% end %>
+    </ul>
+<% end %>
+
+<% if false %>
+<div>
+    <h1>Sign in page</h1>
+     <div>
+         <form accept-charset="UTF-8" action="/users" method="post">            
+             <label for="username">Username:</label><br>
+             <input id="username" name="user[username]" type="text"><br>
+             
+             <label for="email">Email:</label></br>
+             <input id="email" name="user[email]" type="email"><br>
+ 
+             <label for="password">Password:</label><br>
+             <input id="password" name="user[password]" type="password"><br>
+ 
+             <input name="commit" type="submit" value="Create">
+             <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+         </form>
+     </div>
+ </div>
+ <% end %>
+
+ <% if false %>
```

Comment:
This is a smart way of making the code void instead of commenting it out! 

I accept this! 👍